docset/winserver2025-ps/smbshare/Revoke-SmbClientAccessToServer.md

```diff
@@ -0,0 +1,262 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbClientAccessToServer.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 02/22/2024
+online version: https://learn.microsoft.com/powershell/module/smbshare/revoke-smbclientaccesstoserver?view=windowsserver2025-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Revoke-SmbClientAccessToServer
+---
+
+# Revoke-SmbClientAccessToServer
```

Comment:
I want to add better explanation of revoke vs grant vs block vs unblock, will get back to you